packages/connect-explorer/src/js/components/Response.tsx

```diff
@@ -0,0 +1,226 @@
+import React from 'react';
+import styled, { createGlobalStyle } from 'styled-components';
+import Inspector from 'react-inspector';
+import stringifyObject from 'stringify-object';
+import type { AppState } from '../types';
+
+import * as methodActions from '../actions/methodActions';
+import { useActions } from '../hooks';
+
+const copy = data => {
```

Comment:
this is great candidate for adding to utils

Reply:
yep, I think there is similar code in suite also. adding a todo comment for now.

Reply:
@mroz22 I created issue https://github.com/trezor/trezor-suite/issues/4786 to follow up new candidates for utils package, it would be great if you add it there.